src/assets/sass/components/_chart.scss

```diff
@@ -30,6 +30,30 @@ ccl-chart {
     padding-top: 1.5rem;
   }
 
+  .chart-legend {
+    padding: 2rem;
+    padding-top: 0;
+    color: #757575;
+    font-size: 11px;
+
+    .chart-legend-line {
+      color: #1E8CEC;
+    }
+
+    .chart-legend-icons {
+       padding-bottom: 0.5rem;
+    }
+
+    .chart-legend-minmax {
+      color: darken(#EFF4F7, 10%);
+
+      &:before {
+        padding-left: 0.5em;
+        content: '\2B1B'; // unicode large filled square
```

Comment:
Ugh. I was hoping this would work, but no go on Chrome:

![screen shot 2017-08-03 at 10 01 58](https://user-images.githubusercontent.com/1818302/28925443-eb19383e-7832-11e7-9ae3-7e951333678b.png)

Red == the actual position of the span (by setting `background-color`)

![screen shot 2017-08-03 at 10 01 49](https://user-images.githubusercontent.com/1818302/28925442-eb13419a-7832-11e7-8751-a04e19cb77d6.png)

It looks like Chrome isn't aware of this unicode character? "Black medium square" `\25FC` works fine, so does "upwards double arrow" `\21D1`.

An alternative is to give the `.chart-legend-minmax` a fixed width/height and set the background color rather than color. That looks something like:
```css
.chart-legend-minmax {
    background-color: #cddde6;
    width: 1rem;
    height: 1rem;
    display: inline-block;
    margin-left: 0.5rem;
}
```

Maybe @jfrankl has an idea as to why that specific unicode character doesn't work? Maybe its not a part of Chrome's default font family?